### PR TITLE
[SPRV] Sync of AFF in repo with dev

### PR DESCRIPTION
**Pull request by delucapietro**

### CODE REVIEW

**Review by wurzka — COMMENTED** [review]
Thanks for updating the AFF. I have a few concerns about the naming of the components. See my comments.

File: file-formats/sprv/type/zif_aff_sprv_v1.intf.abap [hunk 1]
```diff
@@ -6,14 +6,18 @@ INTERFACE zif_aff_sprv_v1
     "! <p class="shorttext">Service Provider</p>
     "! Service provider details
     BEGIN OF ty_servprovider,
-      "! <p class="shorttext">Name</p>
+      "! <p class="shorttext">Provider Name</p>
       "! Name of the service provider
-      "! $required
-      name   TYPE c LENGTH 30,
-      "! <p class="shorttext">Prefix</p>
+      "! $showAlways
```
[the remark this answers is not in the document]
Comment (delucapietro) [reply file-formats/sprv/type/zif_aff_sprv_v1.intf.abap:11]: The namespace is separated into two fields: Base and Suffix. 
The namespace itself is mandatory but the base is hardcoded so the user can only enter the suffix in the wizard and we concatenate these fields. The concatenated namespace is required but this is made sure programmatically. 
So the field is from my point of view only a "showAlways" for the display view.

File: file-formats/sprv/type/zif_aff_sprv_v1.intf.abap [hunk 2]
```diff
@@ -6,14 +6,18 @@ INTERFACE zif_aff_sprv_v1
     "! <p class="shorttext">Service Provider</p>
     "! Service provider details
     BEGIN OF ty_servprovider,
-      "! <p class="shorttext">Name</p>
+      "! <p class="shorttext">Provider Name</p>
```
[the remark this answers is not in the document]
Comment (delucapietro) [reply file-formats/sprv/type/zif_aff_sprv_v1.intf.abap:9]: The provider name is the name of the provider proxy that is generated during save of the SPRV. 
We do not display the SPRV name as it is also visible in the header of the selected object. 
Using 'name' as a shorttext could be misleading as it is the name of the dependent provider and not of the selected object.

File: file-formats/sprv/type/zif_aff_sprv_v1.intf.abap [hunk 3]
```diff
@@ -6,14 +6,18 @@ INTERFACE zif_aff_sprv_v1
...
       "! Name of the service provider
-      "! $required
-      name   TYPE c LENGTH 30,
-      "! <p class="shorttext">Prefix</p>
+      "! $showAlways
+      prvname TYPE c LENGTH 30,
```
[the remark this answers is not in the document]
Comment (delucapietro) [reply file-formats/sprv/type/zif_aff_sprv_v1.intf.abap:12]: I will change it to the first suggestion:

The provider name is the name of the provider proxy that is generated during save of the SPRV. 
We do not display the SPRV name as it is also visible in the header of the selected object. 
Using 'name' as a shorttext could be misleading as it is the name of the dependent provider and not of the selected object.

File: file-formats/sprv/type/zif_aff_sprv_v1.intf.abap [hunk 4]
```diff
@@ -6,14 +6,18 @@ INTERFACE zif_aff_sprv_v1
...
+      "! <p class="shorttext">Namespace</p>
+      "! Namespace of the service provider
+      "! $showAlways
+      prvns TYPE c LENGTH 30,
```
[the remark this answers is not in the document]
Comment (delucapietro) [reply file-formats/sprv/type/zif_aff_sprv_v1.intf.abap:16]: Changed to namespace

**Comment by delucapietro** [comment]
Suggestions fixed with pull request #566

**Review by schneidermic0 — APPROVED** [review]
We can still change this object type incompatible. It hasn't been available, yet.

Looks good to me. Thanks @delucapietro for the update.

**Pull request merged**